src/assets/scss/elements/ys-flow-steps.scss

```diff
@@ -28,6 +38,14 @@
       padding-top: rem(42);
     }
 
+    #{$variant-vertical-ref} & {
+      display: flex;
+      padding-top: 0;
+      padding-left: rem(46);

```

Comment:
Hardcoded value and not to the "power of 8" should be 48 if we are to adhere to the grid.